.github/workflows/lint-conventional-commits.yaml

```
@@ -0,0 +1,39 @@
+name: "Lint Conventional Commits"
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}  # Ensure that only one instance of this workflow is running per Pull Request
+  cancel-in-progress: true  # Cancel any previous runs of this workflow
+
+jobs:
+  main:
+    name: Lint Conventional Commits
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5.4.0
+        name: "🤖 Check PR title follows conventional commit spec"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # Have to specify all types because `maint` and `rel` aren't defaults
+          types: |
+            fix
+            feat
+            chore
+            ci
+            docs
+            style
+            refactor
+            perf
+            test
```

```suggestion
            chore
            ci
            docs
            feat
            fix
            perf
            refactor
            style
            test
```
Mind if we alphabetize this list?

Also, the list in `commitlint.config.js` also includes `revert` and `squash`. Should those be here too?